doc/api/training/smd_data_parallel_release_notes/smd_data_parallel_change_log.rst

```diff
@@ -0,0 +1,172 @@
+Sagemaker Distributed Data Parallel 1.2.1 Release Notes
+=======================================================
+
+*Date: June. 29. 2021*
+
+**New Features:**
+
+-  Added support for TensorFlow 2.5.0.
+
+**Improvements**
+
+-  Improved performance on a single node.
+-  Improved performance on small clusters (2-4 nodes).
+
+**Bug fixes**
+
+-  Device selection for SageMaker.
+-  Enable ``sparse_as_dense`` by default for SageMaker distributed data
+   parallel library for TensorFlow APIs: ``DistributedGradientTape`` and
+   ``DistributedOptimizer``.
+
+**Migration to AWS Deep Learning Containers**
+
+This version passed benchmark testing and is migrated to the following AWS Deep Learning Containers:
+
+- TensorFlow 2.5.0 DLC release: `v1.0-tf-2.5.0-tr-py37
+  <https://github.com/aws/deep-learning-containers/releases/tag/v1.0-tf-2.5.0-tr-py37>`__
+
+  .. code::
+
+    763104351884.dkr.ecr.<region>.amazonaws.com/tensorflow-training:2.5.0-gpu-py37-cu112-ubuntu18.04-v1.0
```

Comment:
you can remove that suffix v1.0

Reply:
ok maybe we should have it, since tag 2.5.0-gpu-py37-cu112-ubuntu18.04 will govern all future 2.5 tag, and we may have smddp 1.3 goes into tf 2.5.0

Reply:
For smddp 1.3.0, we will publish another release note for that.